app/service/service.go

```diff
@@ -0,0 +1,75 @@
+package service
+
+import (
+	"log"
+
+	"github.com/TudorHulban/GoLayouter/app/helpers/helpers"
+	"github.com/TudorHulban/GoLayouter/domain/interfaces"
+	"github.com/TudorHulban/GoLayouter/domain/objects"
+)
+
+type Service struct {
+	paths []interfaces.IFileOperations
+}
+
+func NewService(content []string) *Service {
+	var res []interfaces.IFileOperations
+
+	for _, line := range content {
+		if helpers.TypeofFile(helpers.GetFileName(line)) == "file" {
```

Comment:
we are calling helpers here two times....
could we maybe do a decorator that we should call only one time?